django/librarian/models.py

```diff
@@ -137,10 +136,10 @@ def delete(self, *args, **kwargs):
         self.reset(recreate=False)
         super().delete(*args, **kwargs)
 
-    def process_all(self, force=True):
+    def process_all(self, force=True, user=None):
```

Comment:
This is the kind of changes to "functions called from inside a request that incur costs" that I'd like to avoid.